rachellegelden/Home-Together-Canada
# Htc 364/search result container

# [364](https://github.com/rachellegelden/Home-Together-Canada/issues/364)

## Summary
Decoupled profile card generation and filtering from membersearchcontainer
added helper function limitresults, display no results component + logic
membersearchresults for creating the cards
searchresultscontainer for filtering data/logic
tests for everything
fixed typos in mockdata from added fields 

## Relevant Motivation & Context


## Testing Instructions
Run npm test - all should pass
Run server/client - unreg user confirm invalid user still displays
sign in and return to view that profile cards display - should be 5
inspect code

## Developer checklist prior to opening this pull request:


## Reviewer
